packages/flutter/test/widgets/app_test.dart

```diff
@@ -21,6 +35,67 @@ void main() {
     expect(find.byKey(key), findsOneWidget);
   });
 
+  testWidgets('WidgetsApp can override default key bindings', (WidgetTester tester) async {
+    final TestAction action = TestAction();
```

Comment:
uber-nit: maybe move this declaration to just before the second pumpWidget where it is used the first time? I was confused why the first one wouldn't use it until I read on...

Reply:
Done.